project1/styles.css

```diff
@@ -0,0 +1,113 @@
+/* for my pullquote */
+
+blockquote {
+    margin: 0;
+}
+
+/* for my pullquote */
+
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 40px;
+    line-height: 1.25;
+    text-align: center;
+}
+
+/* for my pullquote */
+
+blockquote {
```

Comment:
You also have a ruleset for `blockquote` on lines 3-5, these should be combined, like so:
```css
blockquote {
    margin: 0 150px;
}
```
or 
```css
blockquote {
    margin-top: 0;
    margin-right: 150px;
    margin-bottom: 0;
    margin-left: 150px;
}
```